src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -27,13 +27,19 @@ public static class DocumentDBSamples
         //   IAsyncCollector<T>
         //   ICollector<T>
         public static void InsertDocument(
-            [TimerTrigger("00:01")] TimerInfo timer,
-            [DocumentDB("ItemDb", "ItemCollection")] out ItemDoc newItem)
+            [TimerTrigger("00:01", RunOnStartup = true)] TimerInfo timer,
+            [DocumentDB("ItemDb", "ItemCollection")] out ItemDoc newItem,
```

Comment:
This is a fine thing to test. 


---
In reply to: [63894581](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63894581) [](ancestors = 63894581,63893134)